CONTRIBUTING.md

```diff
@@ -0,0 +1,2 @@
+# Usage steps are located in README.md at the root of this repository
+# Current Node version is located in the .travis.yaml file located in the root of this repository
```

Comment:
I think it would be better to refer to the `package.json` file for the current node version. The `travis.yaml` config is related to a CI platform that we do not seem to be using anymore for this project.